test/driver_test.rb

```diff
@@ -1,6 +1,6 @@
 require_relative 'test_helper'
 
-xdescribe "Driver class" do
+describe "Driver class" do
   describe "Driver instantiation" do
     before do
       @driver = RideShare::Driver.new(
```

Comment:
Consider moving this driver outside the `Driver instantiation` block so that it can be used for other tests.